editor/post-title/index.js

```diff
@@ -76,6 +89,26 @@ class PostTitle extends Component {
 		this.setState( { isSelected: false } );
 	}
 
+	setFocused( focused ) {
+		this.setState( { hasFocusWithin: focused } );
+		if ( focused ) {
+			this.props.clearSelectedBlock();
+		}
+	}
+
+	onOuterFocus() {
+		clearTimeout( this.blurTimer );
+		this.setFocused( true );
+	}
+
+	onOuterBlur( e ) {
+		const target = e.currentTarget;
+		clearTimeout( this.blurTimer );
+		this.blurTimer = setTimeout( () => {
+			this.setFocused( target.contains( document.activeElement ) );
+		}, 0 );
+	}
+
 	handleClickOutside() {
```

Comment:
We can probably delete this now, right?

Reply:
Yes, `clickOutside` relies on this, so if we're removing it, it's likely become unused.